includes/BasicObjects/Surface.js

```diff
@@ -1,177 +1,455 @@
+if (!RFEM) {
+    throw new Error("This script is only for RFEM, it creates surfaces.");
+}
+
```

Comment:
Could be fine have also prototype for setting of Surface type - I can create surface via Roration. Then I want to set surface type with needed parameters - e.g. Standard with thickness.